backend/associations/views/library.py

```diff
@@ -44,13 +49,71 @@ def get_serializer_class(self):
         return LibrarySerializer
 
 
+class LoanableFilter(FilterSet):
+    status = MultipleChoiceFilter(
+        choices=(
+            ("AVAILABLE", "AVAILABLE"),
+            ("BORROWED", "BORROWED"),
+            ("REQUESTED", "REQUESTED"),
+        ),
+        method="filter_status",
+    )
+
+    class Meta:
+        model = Loanable
+        fields = ("status", "library")
+
+    def filter_status(self, queryset, _, filter):
+        """Filter by AVAILABLE, BORROWED or REQUESTED.
+                                   | The loanable is available | The loanable is borrowed |
+        There are PENDING loans    | REQUESTED                 | Impossible               |
+        There are no PENDING loans | AVAILABLE                 | BORROWED                 |
+        """
+
+        # No filter or every filter.
+        if len(filter) == 0 or len(filter) == 3:
```

Comment:
What if `filter` has 2 invalid values and 1 valid value ?  (ex: `[AVAILABLE, PICHE, CLAC]`

Reply:
It's verified before (see `MultipleChoiceFilter`)